### PR TITLE
fix(YomuMangás): update presence to new version

### PR DESCRIPTION
**Pull request by Vitu2002**
## Description 
- Add site alternative names
- Update descriptions
- Update thumbnail
- Update tags
- Remove uneeded settings
- Remove uneeded pages
- Refactor code

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

- SFW Manga

SFW_MANGA

- NSFW Manga (Show NSFW ON / Image is a placeholder)

NSFW_MANGA_ON

- NSFW Manga (Show NSFW OFF) or Privacy Mode

NSFW_MANGA_OFF

</details>

### CODE REVIEW

File: websites/Y/YomuMangás/metadata.json [hunk 1]
```diff
@@ -5,25 +5,29 @@
...
+		"pt_BR": "YomuMangás, sua biblioteca virtual de mangás, manhwas, manhuas, novels, doujins e hentais."
 	},
-	"url": [
-		"yomumangas.com",
-		"dev.yomumangas.com"
-	],
-	"version": "1.0.3",
+	"url": "yomumangas.com",
+	"version": "1.2.7",
 	"logo": "https://i.imgur.com/RBkqiAg.png",
```
Comment (DarkVillager) [inline websites/Y/YomuMangás/metadata.json:20]: Also please change logo to 512x512

File: websites/Y/YomuMangás/presence.ts [hunk 2]
```diff
@@ -1,106 +1,81 @@
-const presence = new Presence({ clientId: "869377195682983957" }),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+const presence = new Presence({ clientId: "869377195682983957" });
 
 presence.on("UpdateData", async () => {
-	let presenceData: PresenceData = { largeImageKey: "logo" };
+	let presenceData: PresenceData = { largeImageKey: "yomu" };
```
Comment (DarkVillager) [inline websites/Y/YomuMangás/presence.ts:4]: We now use imgur links for images.

File: websites/Y/YomuMangás/presence.ts [hunk 3]
```diff
@@ -1,106 +1,81 @@
...
+			const username =
+				document.querySelector("#premid-user-name")?.textContent || "あれ？";
+			presenceData.details = `Perfil de ${username}`;
+			presenceData.state = pathname.endsWith("/settings")
+				? "Configurando"
+				: "Vizualizando";
+			presenceData.largeImageKey =
+				document.querySelector<HTMLImageElement>("#premid-user-avatar")?.src ||
+				"yomu-figurant";
+			presenceData.smallImageKey = "yomu-sky";
```
Comment (DarkVillager) [inline websites/Y/YomuMangás/presence.ts:62]: Ditto

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/Y/YomuMangás/presence.ts [hunk 4]
```diff
@@ -1,106 +1,81 @@
...
+		} else if (pathname.startsWith("/user")) {
+			const username =
+				document.querySelector("#premid-user-name")?.textContent || "あれ？";
+			presenceData.details = `Perfil de ${username}`;
+			presenceData.state = pathname.endsWith("/settings")
+				? "Configurando"
+				: "Vizualizando";
+			presenceData.largeImageKey =
+				document.querySelector<HTMLImageElement>("#premid-user-avatar")?.src ||
+				"yomu-figurant";
```
Comment (DarkVillager) [inline websites/Y/YomuMangás/presence.ts:61]: Ditto

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/Y/YomuMangás/presence.ts [hunk 5]
```diff
@@ -1,106 +1,84 @@
-const presence = new Presence({ clientId: "869377195682983957" }),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+const presence = new Presence({ clientId: "869377195682983957" });
 
 presence.on("UpdateData", async () => {
-	let presenceData: PresenceData = { largeImageKey: "logo" };
+	let presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/qmIiWoY.png",
+	};
 	const { pathname, href } = window.location,
```
Comment (DarkVillager) [inline websites/Y/YomuMangás/presence.ts:7]:
```suggestion
	const { pathname, href } = document.location,
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by DarkVillager — APPROVED** [review]
LGTM

File: websites/Y/YomuMangás/presence.ts [hunk 6]
```diff
@@ -1,106 +1,84 @@
...
+			if (!isNsfw || (isNsfw && nsfw)) {
+				presenceData.details =
+					document.querySelector("#premid-manga-title")?.textContent || "Obra";
+				presenceData.state = !isChapter
+					? "Perfil da Obra"
+					: isNaN(Number(chapter))
+					? chapter
+					: `Capítulo ${chapter}`;
+				presenceData.largeImageKey =
+					isNsfw || !image ? "https://i.imgur.com/usrs3S5.png" : image;
```
Comment (Bas950) [inline websites/Y/YomuMangás/presence.ts:42]: please use a 512x512 image

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/Y/YomuMangás/presence.ts (hunk 6 above)
Comment (theusaf) [reply websites/Y/YomuMangás/presence.ts:42]: It's the standard currently being used by this repository to use 512x512 images. 512x512 generally has a good quality while being smaller in size compared to the larger 1024x1024

Comment (Vitu2002) [reply websites/Y/YomuMangás/presence.ts:42]: > It's the standard currently being used by this repository to use 512x512 images. 512x512 generally has a good quality while being smaller in size compared to the larger 1024x1024

I understand, thanks

**Review by Bas950 — APPROVED** [review]

**Review by DarkVillager — APPROVED** [review]

**Pull request merged**